app/views/content/landing/how-to-fund-your-teacher-training/_collage.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-full-content">
+      <div class="inset">
+        <p class="col-720">
+          If you train to be a teacher, you could be eligible to receive a
+          scholarship or bursary of up to £29k to help you train.
+        </p>
+
+        <%= render Content::ChecklistCollageComponent.new(
+          checklist: [
+            "Scholarships and bursaries do not have to be paid back.",
```

Comment:
@EllieNodder @philspray 
Should these bullets have full-stops at the end? Looks a bit odd but I don't know what the correct style is

Reply:
I think because there's no lead in line and they're full sentences, then probably? But happy for you to disagree @philspray

Reply:
I agree, they aren't really bullets but full sentences so need the full stops.